cypress/e2e/supervision/orders/create-order.cy.js

```diff
@@ -13,40 +13,40 @@ const createOrder = (orderType, orderSubType, orderDate, optional) => {
       .closest(".fieldset")
       .find("Select")
       .select(orderSubType);
-    cy.get('input[name="courtReference"]').should("not.be.disabled").clear();
+    cy.get('input[name="courtReference"]').should("not.be.disabled").clear({ force: true });
     cy.get('input[name="courtReference"]').type(caseRecNumber);
```

Comment:
Do every single one of these need a force true? I thought it was usually for when we want to target disabled elements etc. According to Cypress forcing is for when `it's not worth trying to "act like a user" to get a robot to do the exact steps a user would to interact with an element.` But I do feel like with this step we're trying to mimic a user's steps?
Would putting a wait x amount of time for an element to be visible work here?

Reply:
Agreed. We shouldn't really be using workarounds unless they are absolutely necessary, and if they are, we should understand what the issue we are trying to solve by doing so.